internal/exec/stages/files/files.go

```diff
@@ -83,10 +88,14 @@ func (s stage) Run(config types.Config) error {
 	}
 
 	// add systemd unit to relabel files
-	if err := s.addRelabelUnit(config); err != nil {
+	if err := s.addRelabelUnit(); err != nil {
 		return fmt.Errorf("failed to add relabel unit: %v", err)
 	}
 
+	if err := s.addRelabelFile(); err != nil {
```

Comment:
A comment stating what we are doing there and how this is related to the `addRelabelUnit` above would help future casual readers.